tests/NCronJob.Tests/NCronJobIntegrationTests.cs

```diff
@@ -330,12 +330,9 @@ public async Task StaticAnonymousJobsCanBeExecutedMultipleTimes()
     [Fact]
     public void AddingJobsWithTheSameCustomNameLeadsToException()
     {
-        ServiceCollection.AddNCronJob(
+        Action act = () => ServiceCollection.AddNCronJob(
```

Comment:
This might be somewhat of a breaking change, but I am fine with it.
Even better that it fails earlier TBU